apparelo/apparelo/utils/utils.py

```diff
@@ -295,3 +309,39 @@ def check_if_same_value_dict_list(dict_list, key):
 
 # things to figureout
 	# - how to separate out different tables for different size sets
+
+def validate_mappings(additional_parts, additional_parts_sizes, additional_parts_colours):
```

Comment:
- [ ] can you name the function specifically.... like "validate_additional_parts_mapping"?
- [ ] add validations where parts are added in size/colour mapping table but not added in the additional parts table (also similarly for cutting stitching)